acceptance-tests/src/test/resources/features/UKDrivingLicence/English/DVA/DVACRIAPI.feature

```diff
@@ -0,0 +1,70 @@
+@drivinglicence_CRI_API
```

Comment:
This probably doesnt need to be in an english file as its got no concept of locales its direct to the api